### DISCUSSION TITLE
What Happened to Locatarr?

### DISCUSSION THREAD

**Discussion by BrenekH** [BrenekH/locatarr#3] [Announcements]
**Hello!** If you've clicked on this, you must be looking for the story of what happened to Locatarr. Well, you're in luck because I have the details.

But first, allow me to provide a brief introduction. My name is Brenek Harrison, a self-hosting connoisseur, technology enthusiast, and occasional programmer. It's purely by chance that I find myself sharing this information, but I believe it's well past time to get the story out there.

### What Happened 🧐

Around Christmas time in 2022, the original maintainer of Locatarr, then only known as rustyshackleford36, deleted their GitHub account and created a new one under the alias WPTK. However, they forgot to create backups of the repositories on their account, resulting in the inability to restore them to the new account.

> **Side note:** Deleting an account is not the appropriate way to rename it. GitHub provides a [mechanism in the user settings](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username) that takes care of migrating the repositories and even redirects the old links to the new name. Never delete your account to change your username; it's more trouble than it's worth.

Reddit user [u/Rayzaj](https://reddit.com/u/Rayzaj) noticed this problem soon after, [posting on r/Softwarr](https://www.reddit.com/r/Softwarr/comments/zwr4nr/what_happened_to_locatarr/) in order to find out what had happened to the repository. Unfortunately, not too many answers were to be found other than "The repository is gone. Check out these forks instead."

Fast-forwarding a couple of weeks, I began to notice unusual activity on my fork in the form of stars from random people. After some investigation, I stumbled upon the aforementioned Reddit post that outlined the issue. In order to find answers, I delved into the Git logs of my fork to find a usable email address that I could use to reach out to rustyshackleford26/WPTK.

It was during the resulting email chain that I learned about the botched rename attempt. I provided some suggestions to WPTK on how to restart the project if they wished, and they mentioned that they would look into resolving it.

### Moving Forward 🚀

It's been 10 months since I received that response, and during that time, attention on my fork has continued to grow without any change in WPTK's repos. With the goal of continuing the Locatarr project, I will be taking on the role of maintainer for the fork that I currently run. [Issues](https://github.com/Locatarr/Locatarr/issues) and [Discussions](https://github.com/Locatarr/Locatarr/discussions) are now open, and pull requests are once again being merged.

~Regarding the fork status of the repository, I have submitted a request to GitHub Support to extract (their term, not mine) the repo into its own fork network. I am now waiting for their response.~ The repository is now in its own network and will continue as an independent project.

Although I may have occasional busy periods with school and work, I am committed to dedicating time to maintaining and improving the project.

**Happy Hosting!**

**Comment by WPTK** [comment]
Hey all! Original owner here. Just found this, and I'm super glad to see it. Learning GitHub at the time, it was a fun project, and definitely wanted to pick it up, but as you all know, sometimes life and everything else gets in the way! 

Huge shoutout to @BrenekH for keeping something like this alive! I will be happy to contribute to locatarr's continued life with a new "owner".
